web/src/page_params.ts

```diff
@@ -82,6 +82,7 @@ export const page_params: {
     needs_tutorial: boolean;
     promote_sponsoring_zulip: boolean;
     realm_add_custom_emoji_policy: number;
+    realm_allow_edit_history: boolean;
```

Comment:
Hm, is this always set? I can't actually find the backend code that sets it.

Reply:
Seems like this has since been changed to `realm.realm_allow_edit_history`, great